src/TLuaInterpreter.cpp

```diff
@@ -248,50 +248,50 @@ QString TLuaInterpreter::dirToString(lua_State* L, int position)
     if (lua_isnumber(L, position)) {
         dirNum = lua_tonumber(L, position);
         if (dirNum <= 0 || dirNum >= 13) {
-            return 0;
+            return QString();
         }
         if (dirNum == 1) {
-            return "north";
+            return QStringLiteral("north");
         }
         if (dirNum == 2) {
-            return "northeast";
+            return QStringLiteral("northeast");
         }
         if (dirNum == 3) {
-            return "northwest";
+            return QStringLiteral("northwest");
         }
         if (dirNum == 4) {
-            return "east";
+            return QStringLiteral("east");
         }
         if (dirNum == 5) {
-            return "west";
+            return QStringLiteral("west");
         }
         if (dirNum == 6) {
-            return "south";
+            return QStringLiteral("south");
         }
         if (dirNum == 7) {
-            return "southeast";
+            return QStringLiteral("southeast");
         }
         if (dirNum == 8) {
-            return "southwest";
+            return QStringLiteral("southwest");
         }
         if (dirNum == 9) {
-            return "up";
+            return QStringLiteral("up");
         }
         if (dirNum == 10) {
-            return "down";
+            return QStringLiteral("down");
         }
         if (dirNum == 11) {
-            return "in";
+            return QStringLiteral("in");
         }
         if (dirNum == 12) {
-            return "out";
+            return QStringLiteral("out");
         }
     }
     if (lua_isstring(L, position)) {
         dir = lua_tostring(L, position);
         return dir;
     }
-    return 0;
+    return QString();
```

Comment:
Yeah, this and a few other things may need to be revisited before _4.0_ so that speed-walking and other direction stuff does (or can be made to) use/detect locale related value as that is one thing _I think_ that we cannot demand "Lua API uses English language for" when it come from or goes _directly_ to the MUD Server - 😒